python-mini-projects/projects/Download_images_from_website/scrap-img.py

```diff
@@ -0,0 +1,59 @@
+from selenium import webdriver
+import requests as rq
+import os
+from bs4 import BeautifulSoup
+import time
+
+# path= E:\web scraping\chromedriver_win32\chromedriver.exe
+path = input("Enter Path : ")
+
+url = input("Enter URL : ")
+
+output = "output"
+
+
+def get_url(path, url):
+    driver = webdriver.Chrome(executable_path=r"{}".format(path))
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**
This line of code might contain a resource leak. Resource leaks can cause your system to slow down or crash.

**Fix**
Consider closing the following resource: *driver*. The resource is allocated by call *webdriver.Chrome*. Execution paths that do not contain closure statements were detected. Close *driver* in a try-finally block to prevent the resource leak.